x/programs/cmd/simulator/cmd/plan.go

```diff
@@ -217,10 +217,7 @@ func (c *runCmd) runStepFunc(
 		if err != nil {
 			return err
 		}
-		response, balance, err := programExecuteFunc(ctx, c.log, db, programAddress, params[1:], method, maxUnits)
-		if err != nil {
-			return err
-		}
```

Comment:
Removed errors that were coming from the vm space from the simulator

Reply:
It's going to be written in the output, see https://github.com/ava-labs/hypersdk/blob/a40ad2eff0b9622ff588fe33f2e58008b9642519/x/programs/runtime/runtime.go#L69-L78

Reply:
I just ended up removing the err at all, it all makes much more sense now

Reply:
Isn't this function used to invoke a program function regardless of the source? What if it's just the runtime?